lib/starknet-jvm.md

```diff
@@ -156,6 +156,10 @@ public class Main {
 
         // Create an account interface
         Felt privateKey = Felt.fromHex("0x123");
+        
+        // You can also generate a random private key
+         Felt randomPrivateKey = StarknetCurve.generatePrivateKey();
+        
```

Comment:
note: Example of generating account with random/own private key could be split into separate subsections. For now, I've added it as follows, but general guide and examples will be improved in https://github.com/software-mansion/starknet-jvm/issues/504.